.github/scripts/create-profile.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+stage=$1
+
+cat <<< "$(jq \
```

Comment:
this fancy cat command is just a update file in place rather than create new file or append to file